middleware/prometheus/metrics.go

```diff
@@ -8,6 +8,15 @@ import (
 )
 
 var (
+	jobLifeTimeSeconds = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "work",
+			Name:      "job_life_time_seconds",
+			Help:      "Time from a job is created to it is finished",
+			Buckets:   []float64{1, 10, 60},
```

Comment:
Speechkit metrics collected for different job types use different bucket sizes.  I'm for making it generic just wanted to call it out